docs/manual/grelfunctions.md

```diff
@@ -56,7 +56,13 @@ You can use toString() to convert numbers to strings with rounding, using an [op
 
 You can also convert dates to strings, using date parsing syntax built into OpenRefine (see [the toDate() function for details](#todateo-b-monthfirst-s-format1-s-format2-)). For example, `value.toString("MMM-dd-yyyy")` would convert the date value [2024-10-15T00:00:00Z] to “Oct-15-2024”.
 
-Note: In OpenRefine, using toString() on a null cell outputs the string “null”.
+:::note
+In OpenRefine, using `toString()` on a null valued cell such as `value.toString()` outputs the string “null”.
+:::
```

Comment:
I think two boxes in the same go is a bit much emphasis. We got the feedback that we use too much of those boxes in the documentation (which I agree with), so I would rather leave the existing note as is.